apps/core/src/components/named-address/NamedAddress.tsx

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { Address } from '@iota/apps-ui-kit';
-import { useGetIotaName } from '../../hooks';
+import { useGetDefaultName } from '../../hooks';
```

Comment:
Actually liked `IotaName` better, cause it sounds more like a special thing then `DefaultName`. Default name can mean anything.

Reply:
Do you like `useGetDefaultIotaName` ?